### PR TITLE
Time-derivative rule implemented for the inverse Laplace transform

### PR DESCRIPTION
**Pull request by hanspi42**
#### References to other Issues or PRs

Part of #24561

#### Brief description of what is fixed or changed

This change implements the time-derivative rule of the inverse Laplace transform (ILT) and one more simple rule such that the ILT now covers everything required for system theory.

#### Release Notes

* integrals
  * Time-derivative rule implemented for the inverse Laplace transform

### CODE REVIEW

**Comment by hanspi42** [comment]
This is the next step of #24561 

I have looked through all the rules in Bateman54 relevant for system theory, and this is the main thing that was still missing. The rest of the Bateman rules comes later in #24561 under " ILT: Complete rules."

**Comment by hanspi42** [comment]
My next point will be PEP8 cleanup of everything Laplace.

File: sympy/integrals/tests/test_laplace.py [hunk 1]
```diff
@@ -271,10 +271,6 @@ def test_laplace_transform():
         s**3*LaplaceTransform(f(t), t, s) - s**2*f(0) -\
             s*Subs(Derivative(f(t), t), t, 0) -\
             Subs(Derivative(f(t), (t, 2)), t, 0)
-    # Issue #7219
-    assert LT(diff(f(x, t, w), t, 2), t, s) ==\
-        (s**2*LaplaceTransform(f(x, t, w), t, s) - s*f(x, 0, w) -\
-         Subs(Derivative(f(x, t, w), t), t, 0), -oo, True)
```
Comment (sylee957) [inline sympy/integrals/tests/test_laplace.py:277]: Why is this test case removed?

Comment (hanspi42) [reply sympy/integrals/tests/test_laplace.py:277]: I have messed up my local file tree. Please wait with reviewing, sorry.

**Comment by hanspi42** [comment]
@sylee957: Now it is OK; I apologize for the additional review work I caused you.

File: sympy/integrals/laplace.py [hunk 2]
```diff
@@ -1132,6 +1132,7 @@ def _frac(f, s):
     _ILT_rules = [
         (a/s, a, S.true, same, 1),
         (b*(s+a)**(-c), t**(c-1)*exp(-a*t)/gamma(c), c>0, same, 1),
+        (1/(s**2+a**2)**2, (sin(a*t) - a*t*cos(a*t))/(2*a**3), S.true, same, 1)
```
Comment (sylee957) [inline sympy/integrals/laplace.py:1135]: I think that this additional rational function rule is added silently though.

File: sympy/integrals/tests/test_laplace.py [hunk 3]
```diff
@@ -439,6 +440,13 @@ def simp_hyp(expr):
         c*(a*exp(b*t) - b*exp(a*t))*exp(-t*(a + b))*Heaviside(t)/(a - b)
     assert ILT(c*s/(d**2*(s+a)**2+b**2), s, t) ==\
         c*(-a*d*sin(b*t/d) + b*cos(b*t/d))*exp(-a*t)*Heaviside(t)/(b*d**2)
+    # Test time_diff rule
+    assert ILT(s**42*f(s), s, t) ==\
+        Derivative(InverseLaplaceTransform(f(s), s, t, None), (t, 42))
```
Comment (sylee957) [inline sympy/integrals/tests/test_laplace.py:445]: The test case looked quite exhaustive for picking `42` as exponent though.

File: sympy/integrals/laplace.py (hunk 2 above)
Comment (hanspi42) [reply sympy/integrals/laplace.py:1135]: Ah ... almost, I announced it in #24561 but forgot to mention it again here. This is one interesting case that `apart` does not handle.

File: sympy/integrals/tests/test_laplace.py (hunk 3 above)
Comment (hanspi42) [reply sympy/integrals/tests/test_laplace.py:445]: ;)

**Pull request merged**